Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -705,23 +888,30 @@ namespace OpenInfraPlatform {
 					return true;
 				}
 
+				/*! \brief Triangulates merged 2D and 3D vertices to faces.
+				\param	faceVertices2D to be triangulated using carve
+				\param	faceVertices3D to be added to the triangulation
+				\param	faceLoopReversed to adapt order of adding vertices of a new face
+				\param	polygon to compare triangulated vertices with existing ones in polygon, and only add them if they're new
+				\param	polygonIndices
+				*/
+
 				void triangulateFace(const std::vector<carve::geom::vector<2>>& faceVertices2D,
 					const std::vector<carve::geom::vector<3>>& faceVertices3D,
 					const bool faceLoopReversed,
-					std::shared_ptr<carve::input::PolyhedronData> polygon,
-					std::map<std::string, uint32_t>& polygonIndices) const throw(...)
+					EXPRESSReference<carve::input::PolyhedronData> polygon,
+					std::map<std::string, uint32_t>& polygonIndices)

```

Comment:
previous version was correct. If it isn't compiling, rather add `const` modifiers to functions not having it.

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/151/commits/a0cf345765ef7fda9e34e155f58d3058b5844210